### PR TITLE
Mario/homepage tests

### PR DESCRIPTION
**Pull request by ElTragon**

### CODE REVIEW

File: cypress/integration/publicPaths/render/homepage/index.spec.ts [hunk 1]
```diff
@@ -9,12 +13,67 @@ describe(`${pageName} renders expected components on different devices`, () => {
...
+      newShouldArgs("be.visible.and.contain", "View pricing"),
+    ),
+
+    newExpectationWithScrollIntoView(
+      "should scroll to Steps header",
+      "[data-cy=steps-container]",
+      newShouldArgs("be.visible.and.contain", "Get started in minutes"),
+      true,
+    ),
+
```
Comment (liuliu-dev) [inline cypress/integration/publicPaths/render/homepage/index.spec.ts:39]: could also use a test for the `watch video` button

File: cypress/integration/publicPaths/render/homepage/index.spec.ts [hunk 2]
```diff
@@ -9,12 +13,67 @@ describe(`${pageName} renders expected components on different devices`, () => {
...
+      newShouldArgs("be.visible.and.contain", "Learn about the team"),
+      true,
+    ),
+
+    newExpectationWithScrollIntoView(
+      "should scroll to Compare Header",
+      "[data-cy=compare-container",
+      newShouldArgs("be.visible.and.contain", "Which Dolt is right for you?"),
+      true,
+    ),
```
Comment (liuliu-dev) [inline cypress/integration/publicPaths/render/homepage/index.spec.ts:76]: can add a test for the 3 links in the table.

**Review by liuliu-dev — COMMENTED** [review]
looks good, could add more tests.

**Review by liuliu-dev — APPROVED** [review]

File: cypress/integration/publicPaths/render/homepage/index.spec.ts [hunk 3]
```diff
@@ -9,12 +13,91 @@ describe(`${pageName} renders expected components on different devices`, () => {
...
       "renders container",
-      "[data-cy=homepage-container]",
+      "[data-cy=top-container]",
       newShouldArgs(
         "be.visible.and.contain",
         "The easiest way to build with Dolt.",
       ),
     ),
+    newExpectation(
+      "Should find top button",
```
Comment (tbantle22) [inline cypress/integration/publicPaths/render/homepage/index.spec.ts:23]: Nit, but should be consistent with capitalization (should vs Should)

And better descriptions for this and the one below. "should find launch database button" and "should find pricing link" or something

File: cypress/integration/publicPaths/render/homepage/index.spec.ts [hunk 4]
```diff
@@ -9,12 +13,91 @@ describe(`${pageName} renders expected components on different devices`, () => {
...
+      newShouldArgs("be.visible.and.contain", "AWS RDS"),
+      true,
+    ),
+
+    newExpectationWithScrollIntoView(
+      "should scroll to Features mariaDB link",
+      "[data-cy=features-link-mariaDB]",
+      newShouldArgs("be.visible.and.contain", "MariaDB SkySQL"),
+      true,
+    ),
```
Comment (tbantle22) [inline cypress/integration/publicPaths/render/homepage/index.spec.ts:68]: Do you need to scroll here? Isn't this right next to the aws link?

File: cypress/integration/publicPaths/render/homepage/index.spec.ts [hunk 5]
```diff
@@ -9,12 +13,91 @@ describe(`${pageName} renders expected components on different devices`, () => {
...
+    newExpectation(
+      "should find link to Launch",
+      "[data-cy=compare-container]",
+      newShouldArgs("be.visible.and.contain", "Launch a database"),
+    ),
+
+    newExpectation(
+      "should find link to DoltHub",
+      "[data-cy=compare-container]",
+      newShouldArgs("be.visible.and.contain", "Visit DoltHub"),
```
Comment (tbantle22) [inline cypress/integration/publicPaths/render/homepage/index.spec.ts:99]: Should check for all column links here probably

**Review by tbantle22 — COMMENTED** [review]
Looks good, a few small things

**Review by tbantle22 — APPROVED** [review]

**Pull request merged**